tests/angular/eslint.config.js

```diff
@@ -0,0 +1,59 @@
+import { fileURLToPath } from 'url';
+import path from 'path';
+
+import angular from '@ni/eslint-config-angular';
+import angularTemplate from '@ni/eslint-config-angular/template';
+import angularRequiringTypeChecking from '@ni/eslint-config-angular/requiring-type-checking';
+import { ignoreAttributes } from '@ni/eslint-config-angular/template/options';
+
+import typescriptPlugin from 'typescript-eslint';
+import angularTemplatePlugin from 'angular-eslint';
+import { defineConfig } from 'eslint/config';
+
+const tsconfigRootDir = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
```suggestion
const tsConfigRootDir = path.dirname(fileURLToPath(import.meta.url));
```
Should this be renamed?